content/docs/notebooks/setup.md

```diff
@@ -59,7 +59,7 @@ getting-started guide for your chosen environment. For example:
 
 ## Create a Jupyter notebook server and add a notebook
 
-1. Click **Notebook Servers** in the left-hand panel of the Kubeflow UI to 
+1. Click **Notebooks** in the left-hand panel of the Kubeflow UI to 
```

Comment:
Thanks! Is this a change in Kubeflow v0.6.2?

If we change the text, we should change the screenshots too:
https://deploy-preview-1134--competent-brattain-de2d6d.netlify.com/docs/notebooks/setup/

Reply:
Yes. Committed a new change to replace the old screenshot.